### ISSUE TITLE
convertkit.com

### ISSUE THREAD

**Issue by gammons** [lightswitch05/hosts#23]
Hi there,  I'm the Dir of engineering at Convertkit.com and also run a pi-hole on my local network.  I was curious as to why convertkit.com is blocked.  

We do click/open tracking for emails sent, which is consumed at `el2.convertkit.com`, but that's not on your list.  Could I recommend substituting the wholesale blockage with just blocking `el2.convertkit.com`?  Thanks!

**Comment by lightswitch05** [comment]
Hello @gammons thanks for reaching out to me, I'm happy to discuss why I added this block. I found `cdn.convertkit.com` and `api.convertkit.com` on a website that I visited and noticed it was setting cookies as well as sending `viewed` signals to the API. This seems like tracking to me and as far as I can tell there is no way to opt-out. I would also like to note that by browser is configured to send the 'do not track' signal.

But in this case I'm happy to remove your root domain and the other obviously non-tracking domains. I hope commit https://github.com/lightswitch05/hosts/commit/e55708f195bf12b312595408e1ec38e04619a45b is a happy compromise with this request
